src/main/java/com/paneedah/mwc/items/guns/MP5A5Factory.java

```diff
@@ -21,875 +16,731 @@
 import net.minecraft.util.math.Vec3d;
 import org.lwjgl.opengl.GL11;
 
-import java.util.Arrays;
-
 public class MP5A5Factory {
 
     public Item createGun(CommonProxy commonProxy) {
         return new Weapon.Builder()
+                .withName("mp5a5")
+                .withFireRate(0.8f)
+                .withRecoil(1.7f)
```

Comment:
No balance changes until 0.2.
```suggestion
                .withRecoil(1.5f)
```